pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.554.3</version>
+    <version>1.580.1</version>
```

Comment:
jenkins-admin
22:44 Added amuniz as a GitHub committer for repository analysis-pom-plugin